src/reducers-actions/orders.ts

```diff
@@ -38,21 +79,67 @@ export const reducer = (state: OrdersState, action: ReducerActionType): OrdersSt
         payload: { orders },
       } = action
 
-      // default sorting order is ascending by order id
-      // since we are starting from 0, reverse to have the latest on top
-      // make a copy first (slice(0)) to avoid mutating current state
-      const reversedOrders = orders.slice(0).reverse()
+      const reversedOrders = orders
+        // make a copy first (slice(0)) to avoid mutating current state
+        .slice(0)
+        // default sorting order is ascending by order id
+        // since we are starting from 0, reverse to have the latest on top
+        .reverse()
+        // remove deleted orders
+        .filter(order => !isOrderDeleted(order))
```

Comment:
just filter first, that returns a new array, then reverse

Reply:
Ohhhhh thanks for the tip